values-hub.yaml

```diff
@@ -3,7 +3,7 @@ clusterGroup:
   isHubCluster: true
   # Note: setting this to true stores the vault unseal keys inside a cluster secret and
   # is fundamentally insecure
-  insecureUnsealVaultInsideCluster: false
```

Comment:
Can we split this change out in a separate PR? It's really unrelated to this work